File: Programming-languages-and-laboratory/Java/Exercises/18_compito_java_2017_09_11/Azienda.java
public class Azienda extends Iscritto {
	private String ragioneSociale, nomeCognomeGestore;

	public Azienda(String tipo, int codice, String ragioneSociale, String indirizzo, String nomeCognomeGestore,
			String nomeAccount, String password) {
		super(tipo, codice, indirizzo, nomeAccount, password);
		this.ragioneSociale = ragioneSociale;
		this.nomeCognomeGestore = nomeCognomeGestore;
	}

	public String getNome() {
		return ragioneSociale;
	}

	public String toString() {
		return codice + "\t-\t" + ragioneSociale + "\t" + nomeCognomeGestore + "\t" + super.toString();
	}

	public String toStringFollowerNumber() {
		return codice + "\t" + ragioneSociale + "\t" + seguaci.size();
	}
}

File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/18_compito_java_20170911/Azienda.py
from Iscritto import Iscritto


class Azienda(Iscritto):
    def __init__(self, tipo, codice, ragione_sociale, indirizzo, nome_cognome_gestore, nome_account, password):
        super().__init__(tipo, codice, indirizzo, nome_account, password)
        self._ragione_sociale = ragione_sociale
        self._nome_cognome_gestore = nome_cognome_gestore

    def get_nome(self):
        return self._ragione_sociale

    def __str__(self):
        return str(self._codice) + "\t-\t" + self._ragione_sociale + "\t" + self._nome_cognome_gestore + "\t" + self._indirizzo + "\t" + self._nome_account + "\t" + str(len(self._password))
